### DISCUSSION TITLE
event dates and certainty

### DISCUSSION THREAD

**Discussion by bwbohl** [music-encoding/music-encoding#1466] [Q&A] [answered]
I want to indicate that the assumed date of an event is uncertain, do you think the following is clear about that?

```xml
<event isodate="1970-04" cert="low" />
    <!-- … -->
```

Or would you prefer the following?

```xml
<event>
    <date isodate="1970-04" cert="low" />
    <!-- … -->
```

Thanks!

**Comment by rettinghaus** [comment, marked as answer]
I would argue that in the first example the certainty that there was an event is low, whereas the second one more clearly states, that just the date is unclear.

**Reply by bwbohl** [reply]
thanks that is also my impression
